src/GafferArnoldUI/ArnoldLightVisualiser.cpp

```diff
@@ -44,16 +44,20 @@
 #include "IECoreScene/Shader.h"
 #include "IECoreScene/ShaderNetworkAlgo.h"
 
+#include "IECoreGL/CurvesPrimitive.h"
+
 #include "IECore/MessageHandler.h"
 #include "IECore/Exception.h"
 #include "IECore/TypedData.h"
+#include "IECore/VectorTypedData.h"
 
 #include "OSL/oslquery.h"
 
 #include "boost/algorithm/string/predicate.hpp"
 
 using namespace Imath;
 using namespace IECore;
+using namespace IECoreGLPreview;
```

Comment:
None of these changes seem to be necessary - did they get left behind after moving the visualisation to StandardLightVisualiser?

Reply:
Good spot, thanks, sorry about that. https://github.com/GafferHQ/gaffer/pull/3533/commits/93f35d8e86050b362b4a55629a6851a019bdd2d7